### PR TITLE
Upgrade to upstream version 1.2.12

### PR DESCRIPTION
**Pull request by JimboJoe**
## Problem
- *Th application is not up-to-date with the upstream*

## Solution
- *Upgrade to latest version 1.2.12*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u, frju365
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : frju365
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/kanboard_ynh%20PR88/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by JimboJoe** [comment]
:up:

**Comment by frju365** [comment]
Could you please check the SUM of the file, I don't obtain the same as in the PR. ;)

File: conf/app.src [hunk 1]
```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/kanboard/kanboard/archive/v1.2.10.tar.gz
-SOURCE_SUM=4368f0d5902a4831642103fdab90d2a61b6987ce77b046e637883168e4ad0bcb
+SOURCE_URL=https://github.com/kanboard/kanboard/archive/v1.2.12.tar.gz
+SOURCE_SUM=e1163e339dd61ace627259ed40376a5b51c4be0a94a340cdf4489d43f3e5a623
```
Comment (frju365) [inline conf/app.src]: I obtain the same sum. I obtain : 6b17a4cfcf2402e69e49826b248c284f5ed84b02aaf054743c2a46f4af2f9fa3

Comment (frju365) [reply conf/app.src]: Perhaps it's me, so if you could check ;)

**Comment by JimboJoe** [comment]
Thanks for review, I just updated the checksum...
We usually have to wait after a kanboard release because the archive is regenerated later with the vendor dependencies (what I did).
But here the archive changed once again afterwards...

**Comment by frju365** [comment]
Ok. Thanks :-) just one final review of @kay0u and we will merge ;)

**Pull request merged**